Makefile

```diff
@@ -5,7 +5,7 @@ RUST_VERSION ?= $(error RUST_VERSION not set)
 PROJ_VERSION ?= $(error PROJ_VERSION not set)
 DOCKER_TAG=proj-$(PROJ_VERSION)-rust-$(RUST_VERSION)
 
-DOCKER_BUILD_CMD=docker build --build-arg RUST_VERSION=$(RUST_VERSION) --build-arg PROJ_VERSION=$(PROJ_VERSION)
+DOCKER_BUILD_CMD=docker build --platform linux/amd64 --build-arg RUST_VERSION=$(RUST_VERSION) --build-arg PROJ_VERSION=$(PROJ_VERSION)
```

Comment:
Oh nice - is this sufficient to build all our containers on aarch64? I've always just done it on an x86 machine, but it'd be nice to have the option to build on a mac.

Reply:
Yep!